commit ad968f23251c25b4bd391a817d7b60ac0d947273
Merge: 4e84c80 9c5b462
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Thu Jul 7 12:31:15 2016 -0700

    Merge pull request #4234 from thinkyhead/rc_better_position_logging
    
    Move XYZ in print_xyz and DEBUG_POS to the front

diff --git a/Marlin/Marlin_main.cpp b/Marlin/Marlin_main.cpp
index 836b543..61194fd 100644
--- a/Marlin/Marlin_main.cpp
+++ b/Marlin/Marlin_main.cpp
@@ -573,19 +573,19 @@ void serial_echopair_P(const char* s_P, unsigned long v) { serialprintPGM(s_P);
 static void report_current_position();
 
 #if ENABLED(DEBUG_LEVELING_FEATURE)
-  void print_xyz(const char* prefix, const float x, const float y, const float z) {
-    SERIAL_ECHO(prefix);
-    SERIAL_ECHOPAIR(": (", x);
+  void print_xyz(const char* suffix, const float x, const float y, const float z) {
+    SERIAL_ECHOPAIR("(", x);
     SERIAL_ECHOPAIR(", ", y);
     SERIAL_ECHOPAIR(", ", z);
-    SERIAL_ECHOLNPGM(")");
+    SERIAL_ECHOLNPGM(") ");
+    SERIAL_ECHO(suffix);
   }
-  void print_xyz(const char* prefix, const float xyz[]) {
-    print_xyz(prefix, xyz[X_AXIS], xyz[Y_AXIS], xyz[Z_AXIS]);
+  void print_xyz(const char* suffix, const float xyz[]) {
+    print_xyz(suffix, xyz[X_AXIS], xyz[Y_AXIS], xyz[Z_AXIS]);
   }
   #if ENABLED(AUTO_BED_LEVELING_FEATURE)
-    void print_xyz(const char* prefix, const vector_3 &xyz) {
-      print_xyz(prefix, xyz.x, xyz.y, xyz.z);
+    void print_xyz(const char* suffix, const vector_3 &xyz) {
+      print_xyz(suffix, xyz.x, xyz.y, xyz.z);
     }
   #endif
   #define DEBUG_POS(PREFIX,VAR) do{ SERIAL_ECHOPGM(PREFIX); print_xyz(" > " STRINGIFY(VAR), VAR); }while(0)
